src/components/label/Label.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { StyleSheet, View, ViewStyle } from 'react-native';
+import { useTheme } from '../../styles/themes';
+
+import { Body } from '../typography/Body';
+
+interface Props {
+    style?: ViewStyle;
+    text: string;
+    borderColor?: string;
+    textColor?: string;
+    bgColor?: string;
+}
+
+export const Label = ({ text, borderColor, textColor, bgColor, style }: Props) => {
```

Comment:
We could maybe handle this component the same way as the button, with one specific component for each variant (outline, soft and filled) ?
And for the color, an enum of all primary, secondary etc... choices ?